audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/request/AuditRequest.java

```diff
@@ -42,17 +42,14 @@
 import static java.util.Objects.requireNonNull;
 
 /**
- * This is the abstract class that is passed to the audit-service
- * to be able to store an audit record. The default information is
+ * This is the abstract class that is passed to the audit-service to be able to store an audit record. The default information is
  * when was the audit record created and by what server.
```

Comment:
```suggestion
 * This is the abstract class that is passed to the audit-service to be able to store an audit record. The default information consists of
 * when the audit record was created and which server created it.
```
This change makes it easier to read.